src/classes/BDI_DataImport_TEST.cls

```diff
@@ -357,18 +357,29 @@ public with sharing class BDI_DataImport_TEST {
         ID ApexJobId = Database.executeBatch(bdi, 10);
         Test.stopTest();
     
+        Boolean usingAdv = ADV_PackageInfo_SVC.useAdv();
         // verify expected results
         List<Contact> listCon = getContacts();
-        System.assertEquals(1, listCon.size());
+        Integer numCons = usingAdv ? 2 : 1;
+        System.assertEquals(numCons, listCon.size());
         System.assertEquals('TestFirstname00 TestLastname00', listCon[0].Name);
 
         listDI = getDIs();
         System.assertEquals(1, listDI.size());
-        System.assertEquals(null, listDI[0].Contact1Imported__c);
-        System.assertEquals(null, listDI[0].Contact2Imported__c);
-        System.assertEquals(null, listDI[0].HouseholdAccountImported__c);
-        System.assertEquals(label.bdiErrorNonHHAccountContact, listDI[0].Contact1ImportStatus__c);
-        System.assertEquals(null, listDI[0].Contact2ImportStatus__c);
+        // the outcome will be different if Advancement is installed because non-household Accounts CAN be donors
```

Comment:
I don't think this comment matches what this test is doing. Also, non-household accounts can be donors in NPSP.